specs/src/specs/data_square_layout.md

```diff
@@ -37,16 +36,16 @@ Specifically, blobs must begin at a new share. We note a nice property from this
 
 This, however, requires the block producer to interact with the transaction sender to provide them the starting location of their blob, so that the sender can sign over the commitment based on that starting location. This can be done selectively, but is not ideal as a default for e.g. end-user wallets.
 
-### Non-Interactive Default Rules
+### Non-Interaction Rules
```

Comment:
```suggestion
### Share Commitment Rules
```
or Blob Commitment Rules

Reply:
or Blob Share Commitment Rules 🙂

Reply:
Yeah I'm a fan of that

Reply:
cc also curious what @adlerjohn thinks about the name

we've changed them to "Blob Share Commitment Rules" for now. Personally, I'd opt for something like "Blob Layout Rules" or just "Blob Commitment Rules", but bscr are suitable as well

Reply:
I'm not too picky though, whatever people can get behind I'm good with